src/sentry/integrations/slack/views/link_team.py

```diff
@@ -89,18 +90,19 @@ def handle(self, request: Request, signed_params: str) -> HttpResponse:
         organization_memberships = OrganizationMember.objects.get_for_integration(
             integration, request.user
         )
-        # Filter to organizations where we have sufficient role.
-        organizations = [
-            organization_membership.organization
-            for organization_membership in organization_memberships
-            if is_valid_role(organization_membership)
-        ]
-
-        teams_by_id = {
-            team.id: team
-            for organization in organizations
-            for team in Team.objects.get_for_user(organization, request.user)
-        }
+        # Filter to teams where we have access to, either through having a sufficient organization
```

Comment:
nit: Filter to teams where we have write (admin?) access to

Reply:
changed to `where we have write access to,`